src/views/Home/hooks/useGetTopFarmsByApr.tsx

```diff
@@ -25,7 +25,7 @@ const useGetTopFarmsByApr = (isIntersecting: boolean) => {
   useEffect(() => {
     const fetchFarmData = async () => {
       setFetchStatus(FetchStatus.FETCHING)
-      const activeFarms = nonArchivedFarms.filter((farm) => farm.pid !== 0 && farm.multiplier !== '0X')
+      const activeFarms = nonArchivedFarms.filter((farm) => farm.pid !== 0)
```

Comment:
The check is not needed since nonArchivedFarms is a static list that doesn't contain multiplier data